x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/action/EqlActionIT.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.eql.action;
+
+import org.elasticsearch.Build;
+import org.elasticsearch.action.bulk.BulkRequestBuilder;
+import org.elasticsearch.action.bulk.BulkResponse;
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.action.support.WriteRequest;
+import org.elasticsearch.common.xcontent.XContentType;
+import org.elasticsearch.search.SearchHit;
+import org.junit.BeforeClass;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+import static org.elasticsearch.test.StreamsUtils.copyToStringFromClasspath;
+import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
+import static org.hamcrest.Matchers.equalTo;
+
+public class EqlActionIT extends AbstractEqlIntegTestCase {
+
+    @BeforeClass
+    public static void checkForSnapshot() {
+        assumeTrue("Only works on snapshot builds for now", Build.CURRENT.isSnapshot());
+    }
+
+    public void testEqlSearchAction() throws Exception {
+        final String indexPrefix = "endgame";
+        final String testIndexName = indexPrefix + "-1.4.0";
+        final String testIndexPattern = indexPrefix + "-*";
+
+        String endgame = copyToStringFromClasspath("/endgame.json");
+        endgame = endgame.replace("[index_pattern_placeholder]", testIndexPattern);
+
+        assertAcked(client().admin().indices().preparePutTemplate(testIndexName)
+            .setSource(endgame.getBytes(StandardCharsets.UTF_8), XContentType.JSON).get());
+
+        // Insert test data
+        InputStream is = EqlActionIT.class.getResourceAsStream("/endgame.dat");
+        BulkRequestBuilder bulkBuilder = client().prepareBulk();
+        try (BufferedReader reader = new BufferedReader(
+            new InputStreamReader(is, StandardCharsets.UTF_8))) {
+
+            String line;
+            while ((line = reader.readLine()) != null) {
+                bulkBuilder.add(new IndexRequest(testIndexName).source(line.trim(), XContentType.JSON));
+            }
+            BulkResponse response = bulkBuilder.setRefreshPolicy(WriteRequest.RefreshPolicy.IMMEDIATE).get();
```

Comment:
Import static for `IMMEDIATE`?

Reply:
Seems like this is consistent with 100+ other places 
![Screen Shot 2020-02-18 at 2 00 58 PM](https://user-images.githubusercontent.com/872351/74768741-1b303d00-5257-11ea-91cc-18533d3ba4ae.png)